charts/hdfs-namenode-k8s/values.yaml

```diff
@@ -87,3 +84,17 @@ kerberosConfigFileName: krb5.conf
 # where HOST-NAME should match the cluster node
 # host name that each per-host hdfs principal is associated with.
 kerberosKeytabsSecret: hdfs-kerberos-keytabs
+
+# Node labels and tolerations for pod assignment
+nodeSelector: {}
```

Comment:
I wonder if we can add a comment here with the example of the pinning. i.e.
```
# It's possible to pin namenode pods to a set of nodes using nodeSelector.
# You can label your nodes like:	
#   $ kubectl label nodes YOUR-NODE hdfs-namenode-selector=hdfs-namenode
# Then specify nodeSelector like:	
# nodeSelector:	
#   hdfs-namenode-selector: hdfs-namenode
```

Reply:
I believe that for pro users it should be obvious what namenodeSelector is. For others, I fixed namenode readme.

Reply:
Fair enough. Thanks!